internal/model/datacenter.go

```diff
@@ -99,3 +99,19 @@ func (dc *Datacenter) GetPerfMetrics(ref mor) []performance.PerfMetric {
 	}
 	return nil
 }
+
+func (dc *Datacenter) GetDatastore(ds mor) *mo.Datastore {
```

Comment:
same as before, someone using this function risk to have a nil pointer without expecting it